keymaps/project-viewer.json

```diff
@@ -1,7 +1,16 @@
 {
-  ".platform-darwin, .platform-win32, .platform-linux": {
```

Comment:
Previously, these keybindings changes occurred due to this issue [https://github.com/jccguimaraes/atom-project-viewer/issues/46#issuecomment-225744073](url).

Won't these new changes affect again what was discussed here?

Also, making another change in the bindings, will people that are using like it?